app.amsterdam.json

```diff
@@ -0,0 +1,129 @@
+{
+  "apiBaseUrl": "https://acc.api.data.amsterdam.nl",
+  "areaTypeCodeForDistrict": "stadsdeel",
+  "featureFlags": {
+    "assignSignalToDepartment": false,
+    "assignSignalToEmployee": false,
+    "fetchDistrictsFromBackend": false,
+    "fetchQuestionsFromBackend": false,
+    "mapFilter24Hours": true,
+    "showVulaanControls": true,
+    "useStaticMapServer": true
+  },
+  "head": {
+    "androidIcon": "/icon_192x192.png",
+    "backgroundColor": "#ffffff",
+    "favicon": "/favicon.png",
+    "iosIcon": "/icon_180x180.png",
+    "statusBarStyle": "default",
+    "themeColor": "#ec0000"
+  },
+  "language": {
+    "avgDisclaimer": "De gemeente Amsterdam verwerkt uw persoonsgegevens op een zorgvuldige en veilige manier in overeenstemming met de geldende wet- en regelgeving, waaronder de Algemene Verordening Gegevensbescherming. Lees meer hierover op <a href='##AVG_DISCLAIMER_URL##'>Privacyverklaring Meldingen openbare ruimte en overlast</a>.",
+    "district": "Stadsdeel",
+    "consentToContactSharing": "Ja, ik geef gemeente Amsterdam en gemeente Weesp toestemming om mijn contactgegevens te delen met andere organisaties, als dat nodig is om mijn melding goed op te lossen.",
+    "footer1": "Bel het Gemeentelijk informatienummer: <a href='tel:##GENERAL_PHONE_NUMBER_FORMATTED##'>##GENERAL_PHONE_NUMBER##</a>",
+    "footer2": "op werkdagen van 08.00 tot 18.00 uur.",
+    "headerTitle": "",
+    "phoneNumber": "14 020",
+    "shortTitle": "SIA",
+    "siteAddress": "meldingen.amsterdam.nl",
+    "siteTitle": "SIA - Signalen Informatievoorziening Amsterdam",
+    "smallHeaderTitle": "SIA",
+    "title": "Signalen Informatievoorziening Amsterdam",
+    "urgentContactInfo": "Voor spoedeisende zaken kunt u ook telefonisch contact opnemen met 14 020.",
+    "logoDescription": "Logo gemeente Amsterdam, naar amsterdam.nl"
+  },
+  "links": {
+    "help": "https://tamtam.amsterdam.nl/do/office?id=1723860-6f6666696365",
+    "home": "https://www.amsterdam.nl",
+    "privacy": "https://www.amsterdam.nl/privacy/specifieke/privacyverklaringen-wonen/meldingen-overlast-privacy/"
+  },
+  "logo": {
```

Comment:
You can leave the whole `logo` object. Only define it if you need a logo (other than the Amsterdam logo).

Reply:
We need to keep this so the impact on our build street minimal is.